monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -129,6 +130,17 @@ def set_data_source(self, data_src_cfg: str) -> None:
         """
         self.data_list_file = data_src_cfg
 
+    def set_mlflow_tracking_uri(self, mlflow_tracking_uri: str | None = None) -> None:
```

Comment:
The `__init__` is already accepting `mlflow_tracking=_uri`, do we really need this method here? I believe we need either this or that (the same for AutoRunner), what do you think?

Reply:
I think that the modification of more parameters of mlflow would be better.

Reply:
But we need to define the scope of the parameters to cover in the first place.

Reply:
Having this function follows conventions as other similar functions in `BundleAlgo` and `BundleGen`, for example, `set_data_stats`.

Reply:
I agree with you and I like this method but I believe we don't need the redundancy of having both this method and a parameter in the init method. If we keep this, we may need to remove `mlflow_tracking_uri` from __init__. What do you think?

Reply:
I follow the design of `data_stats_filename` and `data_src_cfg_name` in the same class. @mingxin-zheng could you please share some thoughts here? Thanks!